crates/sui-storage/src/event_store/mod.rs

```diff
@@ -81,14 +84,14 @@ trait EventStore {
         &self,
         events: &[EventEnvelope],
         checkpoint_num: u64,
-    ) -> Result<(), EventStoreError>;
+    ) -> Result<(), SuiError>;
```

Comment:
FWIW, I roughly rmr we talked about not bloating `SuiError` (or maybe it was "not exposing internal errors to clients")

Reply:
Initially I had our own `EventStoreError` but the problem is after integrating the code in multiple places, it became much harder to keep using our custom error because of other places that expect SuiError or needed anyhow::Error, so I decided it was easier to convert everything to using SuiError.